Automattic/wp-calypso
# Adapt webpack-rtl-plugin to our repo structure

### Background

See #47451 


#### Changes proposed in this Pull Request

* Clean up `package.json` and rename plugin
* Update dependencies
* Migrate tests from `jest` to `mocha`
* Use `@automattic/webpack-rtl-plugin` in `calypso-build`

#### Testing instructions

* Checkout trunk and build the project with `yarn build-client-both`. This will produce `public/evergreen` and `public/fallback`. Save those folders somewhere, this is your baseline
* Checkout this branch, install dependencies (`yarn distclean && yarn`), and run `yarn build-client-both` again. Compare the new `public/evergreen` and `public/fallback` with the baselines from the previous step. There should be no changes.
